AdminApi/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "scriptFile": "index.js",
+  "disabled": false,
+  "bindings": [
+    {
+      "authLevel" : "function",
+      "type"      : "httpTrigger",
+      "direction" : "in",
+      "name"      : "req",
+      "methods"   : [ "get", "post", "put" ],
+      "route"     : "adm/v1/{*segments}"
```

Comment:
since this is an API for internal use, versioning is not strictly necessary, I would just publish it as `/admin/...`

Reply:
ok for get rid of versioning, but the path "admin" is reserved for internal use (that was my first try)

Reply:
ok